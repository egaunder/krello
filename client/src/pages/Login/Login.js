import React from 'react'
import LoginForm from './LoginForm/LoginForm'

import './Login.css'

const Login = () => (
  <div className="login">
    <div className="login__sect">
      <h1 className="login__title">Krello Login Form</h1>
      <LoginForm />
    </div>
  </div>
)

export default Login
