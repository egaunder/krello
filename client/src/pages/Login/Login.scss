.login {
  height: 100vh;
  display: grid;
  grid-template-columns: 30% auto 30%;
  grid-template-rows: 20% auto 30%; 
}

.login__sect {
  grid-column-start: 2;
  grid-column-end: 3;
  grid-row-start: 2;
  grid-row-end: 3;
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  align-content: flex-start;
  text-align: center;
}

.login__sect > * {
  flex: 1 100%;
}

.login__title {
  height: 10%;
}

@media (max-width: 414px) {
  .login__sect {
    grid-column-start: 1;
    grid-column-end: 5;
    padding-left: 1.2em;
    padding-right: 1.2em;
  }
}