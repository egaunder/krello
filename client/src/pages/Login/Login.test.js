import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Login from './Login'

describe('Login', () => {
  test('should render correctly', () => {
    const output = shallow(<Login />)
    expect(shallowToJson(output)).toMatchSnapshot()
  })
})
