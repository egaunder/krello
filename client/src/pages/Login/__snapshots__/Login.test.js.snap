// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<div
  className="login"
>
  <div
    className="login__sect"
  >
    <h1
      className="login__title"
    >
      Krello Login Form
    </h1>
    <Connect(ReduxForm) />
  </div>
</div>
`;
